Src/Support/GoogleApis/Apis/[Media]/Upload/ResumableUpload.cs

```diff
@@ -412,7 +391,7 @@ public Task<IUploadProgress> UploadAsync()
 
             try
             {
-                UploadUri = await InitializeUpload(cancellationToken).ConfigureAwait(false);
+                UploadUri = await InitiateSessionAsync(cancellationToken).ConfigureAwait(false);

```

Comment:
Funny thing, the docs for the `IUploadSessionData.UploadUri` property say "The resumable session URI (UploadUri)", so I guess in the triple-slash I'll reference it as both so its searchable, but do the appropriate member renamings.